main.js

```diff
@@ -0,0 +1,219 @@
+import {
+    SERVER_URL_WEATHER,
+    SERVER_URL_FORECAST,
+    SHAPE_IMG,
+    INPUT_CITY,
+    NAME_CITY_SELECTED,
+    SEARCH_BTN,
+    ADD_FAVORITES_BTN,
+    FAVORITES_LIST,
+    FORECAST_DAY,
+    INFO_TEMPERATURE_CONT,
+    MAIN_FORECAST_CONT,
+} from "./constants.js";
+
+import {
+    fetchWeatherData,
+    createSrcIconWeather,
+} from "./fetch.js";
+
+import {
+    favoriteImg,
+    convertKelvinToCelsius,
+    formatTimestampToTime,
+} from "./utils.js";
+
+let favoriteCities = [];
+let currentCity = '';
+
+SEARCH_BTN.addEventListener('click', () => {
+    currentCity = INPUT_CITY.value.trim();
+    if (currentCity !== '') {
+        addCityName(currentCity)
+    } else {
+        return alert("Пожалуйста, введите название города.");
+    }
+});
+
+INPUT_CITY.addEventListener('keypress', (e) => {
+    currentCity = INPUT_CITY.value.trim();
+    if (e.key === 'Enter') {
+        if (currentCity !== '') {
+            addCityName(currentCity)
+        } else {
+            return alert("Пожалуйста, введите название города.");
+        }
+    }
+});
+
+ADD_FAVORITES_BTN.addEventListener('click', () => {
+    const foundCity = favoriteCities.find(el => el.nameCity === currentCity)
+    if (foundCity) {
+        console.log('1234')
+        deleteCity(foundCity.id)
+        favoriteImg(currentCity, favoriteCities)
+    } else {
+        addFavoritesList(currentCity)
+    }
+});
+
+const addCityName = (City) => {
```

Comment:
сделай параметр с маленькой буквы 
```suggestion
const addCityName = (сity) => {
```

Reply:
сделал